### PR TITLE
Hw 1

### PR DESCRIPTION
**Pull request by 6ALLIKA**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -1,14 +1,43 @@
...
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder wordsFromFile = new StringBuilder();
+        Path filePath = Paths.get(fileName);
+        List<String> lines = null;
+        try {
+            lines = Files.readAllLines(filePath);
+        } catch (IOException e) {
+            e.printStackTrace();
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java:26]: don't ignore excepitons. read the checklist

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -1,14 +1,43 @@
...
+        }
+
+        for (String text : lines) {
+            for (String soloText : text.split(" ")) {
+                if (soloText.startsWith("w") || soloText.startsWith("W")) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java:31]: try to solve with a single check and make 'w' constant

File: src/main/java/core/basesyntax/FileWork.java [hunk 3]
```diff
@@ -1,14 +1,43 @@
...
+                    wordsFromFile.append(soloText.toLowerCase()
+                            .replaceAll("[^a-z]", "")).append(" ");
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java:33]: do you really need this `replaceAll`?

File: src/main/java/core/basesyntax/FileWork.java [hunk 4]
```diff
@@ -1,14 +1,43 @@
...
+                }
+            }
+        }
+
+        String[] result = wordsFromFile.toString().split(" ");
+        Arrays.sort(result);
+
+        return wordsFromFile.length() == 0 ? new String[0] : result;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java:41]:
```suggestion
        return result;
```
it should work, I suppose

File: src/main/java/core/basesyntax/FileWork.java (hunk 3 above)
Comment (6ALLIKA) [reply src/main/java/core/basesyntax/FileWork.java:33]: yes? cause we have symbols like . ? etc

File: src/main/java/core/basesyntax/FileWork.java (hunk 4 above)
Comment (6ALLIKA) [reply src/main/java/core/basesyntax/FileWork.java:41]: it would be worked if change some logic

**Pull request closed without merging**